clients/drcachesim/tests/offline-snappy-serial.templatex

```diff
@@ -1,5 +1,5 @@
 Cache simulation results:
 .*
 LL stats:
-    Hits:                           58,699
+    Hits:                           78,312
```

Comment:
(Wow this is a non-trivial change.  The other "legacy" test had very minor changes.  Wondering if this LRU issue caused other noticeable result differences on other workloads.)

Reply:
oh it failed again

Reply:
Actually I think that is an unrelated flaky test...checking

Reply:
It is #3213 which unfortunately has happened before but nobody has reproduced locally.

Reply:
Read log and see some timeout.

Reply:
> Read log and see some timeout.

If you could paste a link to what you are seeing?  Only the Windows test was red.